.github/workflows/ci.yaml

```diff
@@ -22,15 +22,18 @@ jobs:
   build:
     needs: ci
     runs-on: ubuntu-latest
+    env:
+      TRAVIS_BRANCH: ${{ github.head_ref }}
+      TRAVIS_PULL_REQUEST: ${{ github.event.number || 'false' }}
```

Comment:
This one I'm not 100% sure about, because I don't know if github.event.number returns a falsey value like this.

Reply:
The article says it should return false by default, so I'm leaving it like in the article.